js/config.js

```diff
@@ -139,13 +139,25 @@ Shazam.initEditor = function(id, mode) {
             editor.gotoLine(3, 1);
         }
 
-        // Handle JS editing - only for superusers
+        // Handle JS editing - only for superusers & those granted permission
```

Comment:
Minor point here, but I might suggest that what is being determined is whether or not the current user has js editing privs.  So rather than treat is a super user and part of the js_users as two separate things, maybe there should be an initial check to see if they are a js_editor and then, if not, set the editor to readOnly...